Makefile

```diff
@@ -0,0 +1,34 @@
+PATH_TO_INVENTORY=../integreatly_dev/inventories/hosts
+PATH_TO_VAULT_FILE=~/.vp
+OPENSHIFT_MASTER=integreatly-cicd.skunkhenry.com
+TOWER_OPENSHIFT_USERNAME=redacted
```

Comment:
https://github.com/RHCloudServices/integreatly-help/pull/377 To be added to the secret in ci